### DISCUSSION TITLE
Critical Error in wordpress update

### DISCUSSION THREAD

**Discussion by MigrantBirdd** [community/community#52064] [Programming Help]
### Body

Hey guys;

i got critical error in update new version of wordpress, i tried to deactivate all plugins from cPanel for access to admin page but unfortunately it's impossible. if anybody can help it's appreciate and tankful in advance.

### I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

- [X] Yes

**Comment by JosephShown** [comment]
If deactivating all plugins from cPanel didn’t resolve the issue, there are a few other things you can try:

Check the error logs: Your web host should provide access to error logs that can help you identify the cause of the issue. Check these logs for any relevant error messages.

Switch to a default theme: If you’re using a custom theme, try switching to one of the default WordPress themes (such as Twenty Twenty-One) to see if this resolves the issue.

Manually update WordPress: If you’re unable to access the admin page, you can try manually updating WordPress by following the instructions in the [WordPress Codex](https://codex.wordpress.org/Updating_WordPress#Manual_Update).

Contact your web host: If none of these steps resolve the issue, it might be worth reaching out to your web host for assistance. They may be able to help you troubleshoot the problem and get your site back up and running.

**Reply by MigrantBirdd** [reply]
thanks a lot for your helpful mentioned. it's done and works fine./

**Comment by w3beginners** [comment]
If deactivating all plugins from cPanel didn’t resolve the issue, there are a few other things you can try:

Check the error logs: Your web host should provide access to error logs that can help you identify the cause of the issue. Check these logs for any relevant error messages.

Switch to a default theme: If you’re using a custom theme, try switching to one of the default WordPress themes (such as Twenty Twenty-One) to see if this resolves the issue.
get visit a website [w3beginners.com](https://w3beginners.com/)
